Jellyfin.Server/Middleware/PathTrimMiddleware.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+
+namespace Jellyfin.Server.Middleware
+{
+    /// <summary>
+    /// Removes /emby and /mediabrowser from requested route.
+    /// </summary>
+    public class PathTrimMiddleware
```

Comment:
Can we call it something more descriptive? Like `LegacyEmbyRouteRewrite` or something.